pkg/transformers/neosync/neosync_transformer.go

```diff
@@ -88,3 +97,14 @@ func toAnyPtr(strArray []string) *any {
 	strArrayAny := any(strArray)
 	return &strArrayAny
 }
+
+func (t *transformer[T]) handleTransformError(err error) (any, error) {
+	var errSingleChar *errSingleCharName
+
+	err = mapError(err)
+	if errors.As(err, &errSingleChar) {
+		return string(uppercaseLetters[rand.Intn(len(uppercaseLetters))]), nil
```

Comment:
Should we consider using the seed parameter if provided so that there's some level of determinism? Otherwise the behaviour might not align with the transformer settings.